radixdlt-core/radixdlt/src/test/java/com/radixdlt/network/p2p/discovery/PeerDiscoveryTest.java

```diff
@@ -129,4 +143,56 @@ public void when_unexpected_response_then_ban_peer() throws Exception {
             .get()
             .isBanned());
   }
+
+  @Test
+  public void when_get_peers_then_shouldnt_return_private_peers() {
```

Comment:
Very minor, but slight preference for `should_not` here as shouldnt looks weird without the apostrophe.

Reply:
I prefer to use **must** for something that can't happen.